taskvine/src/manager/vine_current_transfers.h

```diff
@@ -8,7 +8,7 @@ See the file COPYING for details.
 #include "uuid.h"
 
 #define VINE_FILE_SOURCE_MAX_TRANSFERS 1
-#define VINE_WORKER_SOURCE_MAX_TRANSFERS 3 // static 1 until if/when multiple transfer ports are opened up on worker transfer server
+#define VINE_WORKER_SOURCE_MAX_TRANSFERS 10 // static 10 until if/when multiple transfer ports are opened up on worker transfer server
```

Comment:
Remove the comment, it no longer makes sense.

Also note that these are not really "max" values but default values that can be changed with `vine_tune`